openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -152,26 +174,38 @@ func (a *managementCPUsOverride) Admit(ctx context.Context, attr admission.Attri
 	}
 
 	if !a.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
-		return admission.NewForbidden(attr, fmt.Errorf("%s node and namespace caches not synchronized", PluginName))
+		return admission.NewForbidden(attr, fmt.Errorf("%s node or namespace or infra config cache not synchronized", PluginName))
+	}
+
+	if a.isSNOCluster == nil {
+		clusterInfra, err := a.infraConfigLister.Get(infraClusterName)
```

Comment:
I'd like a measure of how long until the first pod is created before and after this change.  Our install is likely be to sensitive to this.  It should be fast, but I'd like to know how quickly this resource is available.

Reply:
Will it be enough to check how long takes installation with and without the PR?

Reply:
@deads2k I checked the installer time under the GCP for two jobs that include the PR and first one was ~29m and the second one ~31, and I checked the installer time under the GCP for jobs without the PR(checked 10 jobs), and the installer time always around ~28-~32 minutes